proto/api/v1/payment.proto

```diff
@@ -53,8 +53,8 @@ message PaymentCustomer {
 message Card {
   string brand = 1;
   string country = 2;
-  int64 exp_month = 3;
-  int64 exp_year = 4;
+  google.protobuf.Timestamp exp_month = 3;
```

Comment:
how should a month be expressed as timestamp ?
What is the reasoning for this PR. The former definition looks sane.

Reply:
Ah you are right, I thought stripe was sending a unix time because it looked strange. But the type was just not casted correctly in typescript.